maths/FindMin.py

```diff
@@ -1,11 +1,11 @@
-def main():
-    def findMin(x):
-        minNum = x[0]
-        for i in x:
-            if minNum > i:
-                minNum = i
-        return minNum
+def find_min(nums):
+    min = nums[0]
+    for x in nums:
+        if min > x:
+            min = x
+    return min
 
+def main():
     print(findMin([0,1,2,3,4,5,-3,24,-56])) # = -56
```

Comment:
Please modify this line to match `find_min` as well so that the code would compile.